cotton/api.py

```diff
@@ -128,8 +128,18 @@ def configure_fabric_for_host(name):
     zone_config = get_provider_zone_config()
 
     if env.provisioning:
-        env.key_filename = zone_config['provisioning_ssh_key']
-        env.user = zone_config['provisioning_user']
+        if 'provisioning_ssh_key' in zone_config:
+            env.key_filename = zone_config['provisioning_ssh_key']
+        else:
+            print(yellow("No 'provisioning_ssh_key' defined in zone config"))
+        if 'provisioning_user' in zone_config:
+            env.user = zone_config['provisioning_user']
+        else:
+            print(yellow("No 'provisioning_user' defined in zone config"))
+        if 'provisioning_password' in zone_config:
+            env.password = zone_config['provisioning_password']
+        else:
+            print(yellow("No 'provisioning_password' defined in zone config"))
```

Comment:
These (warning?) prints should either be fatal errors or removed. Could some of them be given useful default values instead?